openstack/nova/templates/console-configmap.yaml

```diff
@@ -2,10 +2,15 @@
 {{- $cell_name := index . 1 }}
 {{- $config := index . 2 }}
 {{- with index . 0 }}
+  {{- if $config.enabled }}
 [serial_console]
+# nova-serialproxy
+serialproxy_host = 0.0.0.0
+serialproxy_port = {{ $config.portInternal }}
+# nova-compute
 enabled = {{ $config.enabled }}
-  {{- if $config.enabled }}
 base_url = https://{{include "nova_console_endpoint_host_public" .}}:{{ .Values.global.novaConsolePortPublic }}/{{ $cell_name }}/serial
+serialproxy_address = $my_ip
```

Comment:
I cannot find that option in Nova. Can you point me to it?

Shouldn't it be up in the `#nova-serialproxy` section of the `[serial_console]` group?

Maybe you mean `proxyclient_address`? Then it looks like it belongs here, yes.

Reply:
you're right, good catch!